### DISCUSSION TITLE
Looking for UI designers!

### DISCUSSION THREAD

**Discussion by nikhilreddydev** [rishikeshreddyakkireddy/designtodevelopment#143] [Ideas]
If you are a passionate designer
1. who is looking for project experience (contribute to our repo) or
2. having impact with your work (handing designs to developers, a real world scenario) or
3. aspiring designer to craft your work?

then come join us to work together.
Even if you're a beginner, come give it a shot.

contact us:  rishikeshreddy00@gmail.com (owner)
                   nikhilreddydev@gmail.com (maintainer)

**Comment by Sahilgupta4103** [comment]
hey!! looking for some UI/UX designing project. As I am a beginner looking forward to work with new people.
I am handy with FIGMA and CANVA

**Comment by nikhilreddydev** [comment]
okay, welcome @Sahilgupta4103 .

can I have a look at your previous designs?

**Reply by Sahilgupta4103** [reply]
Ya sure. But as I said i am new in this field and Don't have many projects. 
Rest are in my repositories

**Reply by Sahilgupta4103** [reply]
https://github.com/Sahilgupta4103/UI-UX-DESIGNS.git

**Reply by nikhilreddydev** [reply]
@Sahilgupta4103  okay, hop into our discord server. https://discord.gg/8gkuufeC
